tasks/buildpackages/make-rpm.sh

```diff
@@ -91,8 +92,29 @@ function build_package() {
     else
         sudo yum install -y bzip2
     fi
-    ./autogen.sh
-    ./configure $(flavor2configure $flavor) --with-debug --with-radosgw --with-fuse --with-libatomic-ops --with-gtk2 --with-nss
+    # autotools only works in jewel and below
+    if [[ "$majorvers" -le 10 ]] ; then
+        ./autogen.sh
+        ./configure $(flavor2configure $flavor) --with-debug --with-radosgw --with-fuse --with-libatomic-ops --with-gtk2 --with-nss
```

Comment:
@rjfd See here in this line, configure is being called, even though it will be called again in the rpmbuild phase. There's a comment in make-deb.sh that explains it.